docs-source/content/samples/simple/domains/model-in-image/prerequisites.md

```diff
@@ -158,14 +158,26 @@ weight: 1
 
    $ unzip imagetool.zip
 
+   $ ./imagetool/bin/imagetool.sh cache deleteEntry --key wdt_latest
+
    $ ./imagetool/bin/imagetool.sh cache addInstaller \
      --type wdt \
      --version latest \
      --path /tmp/mii-sample/model-images/weblogic-deploy.zip
    ```
 
-   These steps will install WIT to the `/tmp/mii-sample/model-images/imagetool` directory, plus put a `wdt_latest` entry in the tool's cache which points to the WDT ZIP file installer. You will use WIT later in the sample for creating model images.
-
+   Note that the image tool `cache deleteEntry` command above does nothing
```

Comment:
image tool -> Image Tool (it's capitalized in the WIT docs)
command above does nothing -> command does nothing

Reply:
Done.